packages/ra-ui-materialui/src/AdminUI.tsx

```diff
@@ -18,20 +19,40 @@ export const AdminUI = ({
     loginPage = Login,
     authCallbackPage = AuthCallback,
     notification = Notification,
+    error: errorComponent,
     ...props
-}: AdminUIProps) => (
-    <ScopedCssBaseline enableColorScheme>
-        <CoreAdminUI
-            layout={layout}
-            catchAll={catchAll}
-            loading={loading}
-            loginPage={loginPage}
-            authCallbackPage={authCallbackPage}
-            {...props}
-        />
-        {createElement(notification)}
-    </ScopedCssBaseline>
-);
+}: AdminUIProps) => {
+    const [errorInfo, setErrorInfo] = useState<ErrorInfo | undefined>(
+        undefined
+    );
+
+    const handleError = (error: Error, info: ErrorInfo) => {
+        setErrorInfo(info);
+    };
```

Comment:
useless here, it's already taken care of by `CoreAdminUI`

Reply:
removed